tools/fvm-bench/src/main.rs

```diff
@@ -48,6 +48,7 @@ struct Args {
 }
 
 fn run() -> anyhow::Result<()> {
+    env_logger::init();
```

Comment:
are we using this logger somewhere? I don't see that we are using any of its log macros

Reply:
We do in the FVM itself but we don't initialize it in the FVM as it's supposed to be initialized in the top-level project. I'm initializing it here to make debugging easier.